bin/src/main.rs

```diff
@@ -120,8 +121,10 @@ async fn main() {
     if config.log.use_k8s_enrichment == K8sTrackingConf::Always
         && PathBuf::from("/var/log/containers/").exists()
     {
-        match K8sMetadata::new().await {
-            Ok(v) => {
+        let node_name = std::env::var("NODE_NAME").ok();
+        match K8sMetadata::new(user_agent, node_name.as_deref()).await {
+            Ok((driver, v)) => {
+                tokio::spawn(driver);
```

Comment:
Is the driver is spawned in a separate thread because it's continuously updating pod metadata in the background?

Reply:
Yeh, it's a future containing the reflector stream. Spawning it onto the tokio runtime means that it will continually be polled which keeps it's store up to date